### PR TITLE
Benchmark pallet-phala

### LINKED ISSUE #116: Benchmark & weighting for Phala pallets
Besides benchmark, we are also really interested to see how many storage IO can be done within a regular block (0.5s execution time)

### PR DESCRIPTION
**Pull request by tolak**
Fixes #116

### CODE REVIEW

**Comment by h4x3rotab** [comment]
How much does it cost for `register_worker` and `sync_worker_message`?

File: pallets/phala/src/tests.rs [hunk 1]
```diff
@@ -634,6 +634,50 @@ fn test_bug_119() {
...
+				claim_online: true,
+				claim_compute: true,
+			},
+			sequence: 0,
+		};
+		let signature = ecdsa_sign(&sk, &work_message);
+		print!("WorkMessage signature: {:?}", signature);
+	});
+}
+*/
```
Comment (jasl) [inline pallets/phala/src/tests.rs:680]: nit: add a new line here

**Comment by jasl** [comment]
LGTM, please add some comment about comments

**Review by jasl — APPROVED** [review]

File: pallets/phala/src/tests.rs (hunk 1 above)
Comment (tolak) [reply pallets/phala/src/tests.rs:686]: > How much does it cost for `register_worker` and `sync_worker_message`?

Under local dev mode, have a simple test, fee of each:
 - register_worker: 255.5264 micro Unit
 - sync_worker_message: 19.6918 micro Unit

**Pull request merged**